src/api/schema/events/mod.rs

```diff
@@ -0,0 +1,32 @@
+mod log;
+
+use crate::sinks::tap::TapController;
+use async_graphql::{Context, Subscription};
+use async_stream::stream;
+use log::LogEvent;
+use std::sync::Arc;
+use tokio::stream::Stream;
+
+#[derive(Default)]
+pub struct EventsSubscription;
+
+#[Subscription]
+impl EventsSubscription {
+    /// A stream of a component's log events
```

Comment:
```suggestion
    /// A stream of a component's log events.
```

Reply:
These doc comments are a little special -- they become public API documentation via the GraphQL interface:

![image](https://user-images.githubusercontent.com/719722/107155484-8eebbe00-6970-11eb-86b2-988f4b2c97b7.png)

We haven't historically added a period to field comments, so I'll leave particular one alone.